### ISSUE TITLE
Refactor

### ISSUE THREAD

**Issue by brendanreardon** [vanallenlab/firecloud-helper#5]
This tool should be refactored into two different scripts for index and clean to be separate. Log files should be kept by default. Furthermore, after looking through Seunghun's workspace, I am thinking that the default behavior should _not_ be to keep all files within directories containing data model folders, this can be refactored to be faster by using pandas instead of list comprehension too. 

In summary,
- Refactor into two scripts: Probably two days with testing
- Change behavior of keeping logs: An hour
- Change behavior of keeping all files within a directory to be an option: An afternoon or morning

**Comment by brendanreardon** [comment]
Furthermore, I should add a feature to fetch currently running submissions and exclude those submission folders from the indexing.

**Comment by brendanreardon** [comment]
https://github.com/vanallenlab/firecloud-helper/commits/master Solved with latest commit
